src/core/shards/wires.cpp

```diff
@@ -1549,6 +1549,8 @@ struct Spawn : public CapturingSpawners {
     capturing = true;
     passthrough = false;
   }
+  ~Spawn() {
+  }
```

Comment:
Is it necessary?

I read on a thread that an explicit empty destructor can prevent inlining.

Reply:
Left from debugging, will remove